notebooks/community/model_garden/model_garden_pytorch_detectron2.ipynb

```diff
@@ -0,0 +1,1237 @@
+{
```

Comment:
<p>general comment, pls in the future use the current notebook template. It will be costly otherwise if this notebook is later moved to official</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1783/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Noted. Will do that next time.